packages/nextjs/src/middleware/ProxyResponseCallback.ts

```diff
@@ -55,6 +55,9 @@ const ProxyResponseCallback: ProxyResCallback<IncomingMessage, NextApiResponse>
         try {
           if (bodyStr && bodyStr.length > 0) {
             const body = JSON.parse(bodyStr);
+            if (body.authResponse) {
+              Object.assign(body, body.authResponse);
```

Comment:
Can we have `authResponse` on other requests?
Consider adding `url === 'signup'` and extract to util function with meaningful name

Reply:
no need for now, this code will be refactored for better request mapping